### DISCUSSION TITLE
How to integrate with existing laravel project? how to setup routes?

### DISCUSSION THREAD

**Discussion by pratikshap31** [filamentphp/filament#3261] [Help]
I have an existing laravel project. In my project, I have set up the admin panel using the https://github.com/exodusanto/Laravel-Admin package. But, it is not supported for the laravel 9 version. So, we are planning to change the package for the admin panel. I have started implementation using your package however I am facing an issue getting the listing page.

Can someone guide me on how to access the route of this package as I am getting 404 or 403 errors?

**Comment by danharrin** [comment]
As discussed on Discord, https://filamentphp.com/docs/2.x/admin/installation#deploying-to-production

**Reply by pratikshap31** [reply]
@danharrin 
can we add custom buttons for custom actions?

**Reply by danharrin** [reply]
Yes, by passing a custom `view()`

**Comment by pratikshap31** [comment]
@danharrin 

Does the package provide an export CSV feature?

**Reply by danharrin** [reply]
We have export plugins that can do that https://filamentphp.com/plugins

**Comment by pratikshap31** [comment]
@danharrin 
is this package support multiple subdomains? as in my project each admin has their own subdomain.
